src/components/TableComponents/CellTextItem/styled.ts

```diff
@@ -0,0 +1,24 @@
+import { Box, Typography } from "@mui/material";
+import styled from "styled-components";
+
+export const CellTextBox = styled(Box)`
+  display: flex;
+  flex-direction: column;
+  gap: 4px;
+`;
+
+export const CellTextMain = styled(Typography)`
+  color: #252733;
+  font-size: 14px;
+  font-weight: 600;
+  line-height: 20px;
+  letter-spacing: 0.2px;
```

Comment:
you remember that I asked you to create a `theme` constants for all sizes and common styles values?)